os/drivers/lcd/lcd_dev.c

```diff
@@ -461,16 +465,21 @@ int lcddev_register(struct lcd_dev_s *dev)
 	lcd_info->pm_domain = pm_domain_register("LCD");
 #endif
 
-	lcd_init_put_image(dev);
 	sem_init(&lcd_info->sem, 0, 1);
 	if (dev->setpower) {
-		ret = dev->setpower(dev, CONFIG_LCD_MAXPOWER);
+		bool is_silent_mode;
+		is_silent_mode = silent_reboot_is_silent_mode();
+		if (!is_silent_mode) {
+			ret = dev->setpower(dev, CONFIG_LCD_MAXPOWER);
+			lcd_init_put_image(dev);
+#ifdef CONFIG_PM
+                (void)pm_suspend(lcd_info->pm_domain);
+#endif
+			silent_reboot_lock();
+		}
```

Comment:
minor, I think, the lcd_init_put_image() is not releated to the power ops.
how about below?
```suggestion
	bool is_silent_mode;
	is_silent_mode = silent_reboot_is_silent_mode();
	if (!is_silent_mode) {
		if (dev->setpower) {
			ret = dev->setpower(dev, CONFIG_LCD_MAXPOWER);
			(void)pm_suspend(lcd_info->pm_domain);
			silent_reboot_lock();
		}
		lcd_init_put_image(dev);
	}
```

Reply:
lcd_init_put_image() now changes mode to video mode so we should only call it when power is turned on in intialization
If power on didn't execute and we call it, then it can create issue
As app will call power on in that case, it will not be in command mode and sending init commands might encounter issue

Reply:
dev->setpower looks mandatory. So, How about chagne to ASSERT(dev->setpower)?